autogpt_platform/backend/schema.prisma

```diff
@@ -510,9 +508,13 @@ model StoreListing {
   StoreListingVersions   StoreListingVersion[]
   StoreListingSubmission StoreListingSubmission[]
 
-  @@index([isApproved])
-  @@index([agentId])
+  // Unique index on agentId to ensure only one listing per agent, regardless of number of versions the agent has.
+  @@unique([agentId])
+  @@index([agentId, owningUserId])
   @@index([owningUserId])
+  // Used in the view query
+  @@index([isDeleted, isApproved])
```

Comment:
These two indices with Boolean values will only be used on a query like this

`Select ... From StoreListing where is deleted = ...`

If you have another column in the filter, it will never be used. So I don't think this will be useful.

Reply:
If `[isDeleted, isApproved]` is indeed the filter column, the index with two columns will override the one below.
So we can safely remove it.